app/org/sagebionetworks/bridge/play/controllers/EmailController.java

```diff
@@ -83,6 +83,20 @@ public Result unsubscribeFromEmail() throws Exception {
             return ok(throwable.getMessage());
         }
     }
+    
+    /**
+     * An URL to which a POST can be sent to set the user's email notification preference to "off". Cannot turn email
+     * notifications back on through this endpoint. This cannot be part of the public API, because MailChimp doesn't
+     * deal with non-200 status codes. The token that is submitted is set in the configuration, and must match to allow
+     * this call to succeed. Subject to change without warning or backwards compatibility.
+     * 
+     * @return
+     * @throws Exception
+     */
+    @Deprecated
+    public Result unsubscribeFromEmailV1() throws Exception {
+        return unsubscribeFromEmailV3();
+    }
```

Comment:
Instead of copy-pasting the comment, you can use a @see javadoc tag to have V1 reference V3. That way, if you ever need to update the comment, you only need to change it in once place. Plus, no one should be using V1 anyway.